helm/fcp-fd-comms/values.yaml

```diff
@@ -54,7 +54,7 @@ postgresService:
   postgresqlHost: fcp-fd-comms-postgres
   postgresPort: 5432
   postgresSchema: public
-  postgresUser: postgres
```

Comment:
After speaking with John W., learned that default values shouldn't be added to the Helm chart e.g.:
```
postgresUser: postgres
```
This is because it does not allow for fast failure and might mask that there's missing values from appConfig/Key Vault. In a situation where a deployment is successful, it might have been successful because it's chosen to use the default value of `postgres` as opposed to the true value, making it harder to identify the issue when trying to send a message and save it into the database for example. For this reason, `postgresUser` is left as blank.